en/views/helpers/paginator.rst

```diff
@@ -489,6 +489,29 @@ contexts (i.e. JavaScript). ::
 
     echo $this->Paginator->generateUrl(['sort' => 'title']);
 
+.. _paginator-helper-multiple:
+
+Paginating Multiple Results
+===========================
+
+If you are :ref:`paginating-multiple-queries` you'll need to set the ``model``
```

Comment:
Using refs in a sentence uses the title of the referenced part of the docs.
I'm not sure if that's really wise. What if the title gets changed?
Wouldn't that change the wording of the sentence here?

Certainly that doesn't happen that much, but why call for trouble?